source/index.html.md.erb

```diff
@@ -342,16 +356,22 @@ Before you begin, you should already have an access token, which you used in the
 		}
 	],
 	"rows": [{                                      // A Row Object
-			"id": 564480076736388,                  // You must use the row Id to make updates
+			"id": 564480076736388,                  // Use the row Id to make updates
 			"rowNumber": 1,
 			"expanded": true,
 			"createdAt": "2017-05-12T16:52:38Z",
 			"modifiedAt": "2017-05-22T20:40:14Z",
-			"cells": [{                             // Each row contains an array of Cells, which have the actual content
-					"columnId": 2517104256673668,   // The column Id can be interpreted by looking at the array of
-                                                    // column definitions above. That tells us this is the "Name" column
+            "cells": [{                             // Each row contains an array of Cells,
+                                                    // which have the actual content
+
+                    "columnId": 2517104256673668,   // The column Id can be interpreted by
+                                                    // looking at the array of column
+                                                    // definitions above. That tells you
+                                                    // this is the "Name" column
+
 					"value": "John Doe",            // Actual cell value
-					"displayValue": "John Doe"      // How the cell value is displayed in the UI
+                    "displayValue": "John Doe"      // How the cell value is displayed
```

Comment:
This comment would be more helpful on the next, numeric cell. value could be 12345 and displayValue could be "12,345"

Reply:
Done